pkg/promclient/promclient.go

```diff
@@ -26,7 +26,6 @@ import (
 	"github.com/gogo/status"
 	"github.com/pkg/errors"
 	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/pkg/labels"
```

Comment:
:+1:  You have to update all the `labels` alias with `promlabels`  or the other way round.

Reply:
Thanks for the review @prateekpandey14 !!